temba/api/v2/serializers.py

```diff
@@ -846,6 +847,9 @@ def get_status(self, obj):
     def get_attachments(self, obj):
         return [a.as_json() for a in obj.get_attachments()]
 
+    def get_metadata(self, obj):
```

Comment:
Do we need to expose this for now? Seems like metadata schema might change over time